sct.py

```diff
@@ -456,6 +456,10 @@ def send_email(test_id=None, email_recipients=None, logdir=None):
     from sdcm.send_email import (GeminiEmailReporter, LongevityEmailReporter,
                                  get_running_instances_for_email_report,
                                  read_email_data_from_file)
+    LOGGER.info(email_recipients)
```

Comment:
1) Not informative enough. Should be something like : `Going to send test run report to: {email_recipients}`
2) Put this after the if

Reply:
@bentsi updated the message and moved after If block